pkg/server/types.go

```diff
@@ -41,5 +41,9 @@ type BackendMapper struct {
 
 // AccessConfig represents the configuration format for cluster access config via backend mode.
 type BackendModeConfig struct {
+	// Time that the object takes from update time to load time
+	LastUpdatedDateTime string `json:"LastUpdatedDateTime"`
```

Comment:
The var name does not tell whethere it expects epoch time or mm-yy format. In a future change this var should be made more readable.